JASP.pri

```diff
@@ -8,12 +8,16 @@ JASP_R_INTERFACE_MINOR_VERSION =  2   # Code changes
 
 JASP_R_INTERFACE_NAME = $$JASP_R_INTERFACE_TARGET$$JASP_R_INTERFACE_MAJOR_VERSION'.'$$JASP_R_INTERFACE_MINOR_VERSION
 
-GITHUB_PAT_DEF=$$(GITHUB_PAT_DEF)						#First check if we set this special env var (on buildbot)
-isEmpty(GITHUB_PAT_DEF): GITHUB_PAT_DEF=$$(GITHUB_PAT)	#Otherwise use the one we are using
-isEmpty(GITHUB_PAT_DEF){
-	error("No GITHUB_PAT or GITHUB_PAT_DEF found!  Follow the steps at https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-building-guide.md$${LITERAL_HASH}github_pat")
-	
-	GITHUB_PAT_DEF="NO_GITHUB_PAT_FOUND"	#To indicate a clearer error
+!exists(/app/lib/*){
```

Comment:
Why do you do this?

Reply:
well, we cannot set a GITHUB_PAT on flatpak, but we also don't need one.

Reply:
we *do* need one, to install modules... I already ran into that error (but only after also missing `gcc` and `gfortran` so I didnt mention it)

Reply:
okay, then I'll look into some other way to do this!

Reply:
I went for the base64 idea, let's hope GitHub won't notice